body.tex

```diff
@@ -24,7 +25,7 @@ \section{Problem Definition}
 \end{figure}
 
 
-The \emph{thresholding} step has traditionally been implemented as a simple $5\sigma$-thresholding, but other auxilliary approaches may wrap this stage to make it \emph{smart}er -- figure~\ref{fig:diagram}
+The \emph{thresholding} step has traditionally been implemented as a simple $5\sigma$-thresholding, but other auxiliary approaches, such as pre-convolution with the PSF, may wrap around this stage to make it smart -- figure~\ref{fig:diagram}
```

Comment:
I think the phrase "smart" here isn't meaningful (and hence isn't helpful).  The statistical justifications for detection in this way are described (for example) [here](https://dmtn-015.lsst.io/#kaiser-coadds) and [here](https://dmtn-015.lsst.io/#detection-maps); the arguments are very similar for image differences.